packages/@ember/-internals/runtime/lib/mixins/array.js

```diff
@@ -278,7 +278,7 @@ const ArrayMixin = Mixin.create(Enumerable, {
 
     ```javascript
     let peopleToMoon = ['Armstrong', 'Aldrin'];
-    
+
```

Comment:
Sorry for the noise, my editor removed the whitespace in this file.

Reply:
It's helpful, because those whitespaces shouldn't be there in the first place!